### ISSUE TITLE
Quick Download Tree as Package: Including Linked Item; SiteCore Root included breaking Sitecore on Install

### ISSUE THREAD

**Issue by sonicdivx** [SitecorePowerShell/Console#1186]
### Expected Behavior

Using Quick Download Tree as Package and including Linked Items the package created should be able to be installed without breaking the SiteCore Root

### Actual Behavior

When Quick Download is used without selecting any linked items, it creates a package that will successfully install. When adding linked items and the package is installed. Sometimes the SiteCore root node is corrupts and all sites disappear or the Site the package is for becomes corrupted and disappears.

### Steps to Reproduce the Problem
Right-click on page node in website.
Select Scripts
Select Packaging
Select Quick Download Tree as Package
Set Package Meta-Data (FYI package name does not get used for Download file - another issue)
Set Installation options (if Set to overwrite it will corrupt SiteCore,   Merge is more likely to corrupt site)
Set Other > Linked Items. (Screenshots are when I included just Images)

 Images included in Package
Path:   package.zip/items/master/sitecore
![image](https://user-images.githubusercontent.com/966481/79760481-2272d600-82ee-11ea-933d-bb68e3ac1532.png)

No Images Included
Path: package.zip/items/master/sitecore
![image](https://user-images.githubusercontent.com/966481/79760560-3f0f0e00-82ee-11ea-8dbd-d0084e99444f.png)

_Please include the version number of SPE and Sitecore._

Sitecore PowerShell Extensions 5.0.0.42513
Sitecore 9.1
Windows PowerShell 5.1

 - [ ] Tested issue with clean install of Sitecore and the latest available version of SPE.
NO
 
 - [ ] Asked questions on the Sitecore Slack Chat channel.
NO
 
 - [ ] Reviewed questions and answers on the Sitecore Stack Exchange.
NO

**Comment by michaellwest** [comment]
Any idea why the root node is included with the linked items?

**Comment by sonicdivx** [comment]
No, at first I thought maybe a bad right-click, this has happened many times. Where as Package Designer does not do this. I am mostly a front-end dev, but our backend team doesn't use this (they use Package Designer). I like quick actions like this.

**Comment by michaellwest** [comment]
Would it be reasonable to exclude the root node and siblings? Not sure anyone would knowingly select them.

Can you click on the root node and then in the ribbon click Links? Should show you referers.

**Comment by sonicdivx** [comment]
Yeah I can check that.  Yeah not sure why someone would want the root node when doing a quick tree download.

**Comment by AdamNaj** [comment]
I can't seem to replicate this issue and I keep using this functionality quite often. As I see the links outside of roots should not be passing the filter that was added in #923. Closing it for now as it's very stale. If still present, please reopen.
